client/src/pages/NewReport/AccuracySection/AccuracySection.tsx

```diff
@@ -0,0 +1,20 @@
+import SectionComponent from 'components/SectionComponent/SectionComponent.tsx';
+import TagButton from 'components/TagButton/TagButton.tsx';
+import { useState } from 'react';
+import SVGIcon from 'components/SVGIcon/SVGIcon.tsx';
+
+const AccuracySection = () => {
+    const [accuracy, setAccuracy] = useState('HIGH');
```

Comment:
Consider constraining type to the union of literals, such as useState<AccuracyType>()